sound/soc/sof/topology.c

```diff
@@ -2150,12 +2144,29 @@ static int sof_complete(struct snd_soc_component *scomp)
 					if (ret < 0)
 						return ret;
 				}
+
```

Comment:
spurious newline?

Reply:
no, there's a variable declaration just before right?

Reply:
the code was modified, the comment is irrelevant now